### DISCUSSION TITLE
Using markdoc for part of a site?

### DISCUSSION THREAD

**Discussion by benguz** [markdoc/markdoc#361] [Q&A] [answered]
I'm hoping to use markdoc as the documentation portion of a site, eg only for /docs, but I'm not very familiar with nextjs. How can I integrate it with vanilla HTML pages?

**Comment by mfix-stripe** [comment, marked as answer]
Hey @benguz, we have an example of how to use Markdoc to generate HTML here: https://markdoc.dev/docs/examples/html
